app/Helpers/database/player-rank.php

```diff
@@ -8,14 +8,13 @@
 use Illuminate\Support\Carbon;
 use Illuminate\Support\Facades\Cache;
 
-function SetUserUntrackedStatus(User $user, int $isUntracked): void
+function SetUserUntrackedStatus(User $user, bool $isUntracked): void
 {
     $user->Untracked = $isUntracked;
+    $user->unranked_at = $isUntracked ? now() : null;
```

Comment:
This is the double-write for manually unranking something. Banning is already handled by https://github.com/RetroAchievements/RAWeb/blob/e5eb5c086b6a1e5307f35aa22388b35d785b32b5/app/Helpers/database/user-permission.php#L168